src/app/payment/paymentMethod/WalletButtonPaymentMethod.tsx

```diff
@@ -204,13 +207,23 @@ class WalletButtonPaymentMethod extends Component<
             method,
         } = this.props;
 
-        if (normalizeWalletPaymentData(method.initializationData)) {
+        if (normalizeWalletPaymentData(method.initializationData) || this.isStoreCreditUsed()) {
```

Comment:
I think this calc is already in the state `checkoutState.data.isPaymentDataRequired`.
```suggestion
        if (normalizeWalletPaymentData(method.initializationData) || !isPaymentDataRequired) {
```